apps/finbrief/app/(marketing)/finbrief.css

```
@@ -186,6 +186,49 @@
   transition-duration: 0.1s;
 }
 
+/* Blue variant — keeps the same shimmer / hover / active effects */
+.btn-metal.btn-metal--blue {
+  background:
+    linear-gradient(0deg, rgba(255, 255, 255, 0.06) 1px, transparent 1px) 0 0 /
```

Stylelint (17.3.0)

[error] Unexpected newline after "/" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)